app/src/main/res/layout/activity_qractivity.xml

```diff
@@ -4,6 +4,46 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="#F6F6F6"
     tools:context=".ui.QR.QRActivity">
 
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+    <androidx.appcompat.widget.Toolbar
+        android:id="@+id/toolbar"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="#FBBC05"
+        app:layout_constraintEnd_toEndOf="parent"
```

Comment:
Make the Title bolder and increase it's size, It looks small now.

Reply:
Do I need to create a styles xml file for Toolbar Title since there isn't any tag for Title text size and bold?

Reply:
Yes, you can create a new style for toolbar.